aws/cognito/lambda.tf

```diff
@@ -2,31 +2,12 @@
 # COGNITO EMAIL SENDER
 ########################
 
-data "archive_file" "cognito_email_sender_code" {
-  type        = "zip"
-  source_dir  = "lambda/cognito_email_sender/dist"
-  output_path = "/tmp/cognito_email_sender.zip"
-}
-
-resource "aws_s3_object" "cognito_email_sender_code" {
-  bucket      = var.lambda_code_id
-  key         = "cognito_email_sender_code"
-  source      = data.archive_file.cognito_email_sender_code.output_path
-  source_hash = data.archive_file.cognito_email_sender_code.output_base64sha256
-}
-
 resource "aws_lambda_function" "cognito_email_sender" {
-  s3_bucket         = aws_s3_object.cognito_email_sender_code.bucket
-  s3_key            = aws_s3_object.cognito_email_sender_code.key
-  s3_object_version = aws_s3_object.cognito_email_sender_code.version_id
-  function_name     = "Cognito_Email_Sender"
-  role              = aws_iam_role.cognito_lambda.arn
-  handler           = "cognito_email_sender.handler"
-  timeout           = 300
-
-  source_code_hash = data.archive_file.cognito_email_sender_code.output_base64sha256
-
-  runtime = "nodejs18.x"
+  function_name = "cognito-email-sender"
+  image_uri     = "${var.ecr_repository_url_cognito_email_sender_lambda}:latest"
```

Comment:
A note here that this `:latest` tag will cause the Terraform to always try to deploy the latest image when it runs.  The case I'm thinking of is if you've manually set the Docker image tag with another workflow step, TF apply will undo that.

If you don't want this behaviour, you can ignore changes to the `image_uri` like so:
https://github.com/cds-snc/terraform-modules/blob/e9ef5d9669ba8d9304f12e2b98f053d6044b61e0/lambda/main.tf#L34-L38